addons/stock/models/stock_picking.py

```diff
@@ -998,8 +999,11 @@ def _pre_action_done_hook(self):
 
         if not self.env.context.get('skip_backorder'):
             pickings_to_backorder = self._check_backorder()
-            if pickings_to_backorder:
-                return pickings_to_backorder._action_generate_backorder_wizard(show_transfers=self._should_show_transfers())
+            # Do not generate a backorder confirmation wizard for the pickings
+            # that are configurated to generate backorders automatically (i.e. force_backorder is True in its picking type)
+            pickings_backorder_to_confirm = pickings_to_backorder.filtered(lambda picking: not picking.picking_type_id.force_backorder)
```

Comment:
I don't get it, you backorder isn't create then (because they are create from the wizard `process`), no ? 

Also you should add a test for this feature and also for the enterprise part (it will avoid to break it by someone else).